gwctl/cmd/root.go

```diff
@@ -26,31 +25,45 @@ import (
 	"github.com/spf13/cobra"
 	"k8s.io/klog/v2"
 
-	"sigs.k8s.io/gateway-api/gwctl/pkg/common"
-	"sigs.k8s.io/gateway-api/gwctl/pkg/policymanager"
 	cmdutils "sigs.k8s.io/gateway-api/gwctl/pkg/utils"
```

Comment:
Nit: Package name doesn't really match path - maybe the scope of the package expanded over time?